tutorial_submission_policy.md

```diff
@@ -0,0 +1,64 @@
+releases are typically authored by the engineers who are developing these features. These tutorials
+are crucial for showcasing the latest advancements in PyTorch.
+
+* **Tutorials from PyTorch Partners:** We welcome tutorials from our partners who are actively
+  collaborating with us to grow the PyTorch ecosystem.
+  
+* **Direct Requests from PyTorch Maintainers:** If a PyTorch maintainer has directly reached out
+  to you to request a tutorial or otherwise, endorsed a tutorial, this indicates a specific need
+  within the community or the project.
+  
+* **Tutorials from the open-source community members:** We accept tutorials from the community
+  members. However, as we can't accept all of them, we have identified the process below.
+  Please read below before sending a PR.
+
+The first three use cases do not require following the submission process described below.
+If your tutorial falls under the fourth category, please read and follow the instructions in
+the **Submission Process For Community-Contributed Tutorials** section.
+
+## Submission Process For Community-Contributed Tutorials
+
+To maintain the quality and relevance of tutorials, community-contributed tutorials must undergo
+a review process. If you are interested in contributing a tutorial, please follow these steps:
+
+1. **Create an issue:**
+   * Open an issue in the pytorch/tutorials repository proposing the new tutorial. Clearly
+     explain the importance of the tutorial and confirm that there is no existing tutorial
+     covering the same or similar topic. A tutorial can't endorse one technology over another.
+   * These issues will be triaged by PyTorch maintainers on a case-by-case basis. 
+   * Link any supporting materials including discussions in other repositories".
+     
+1. **Await Approval:**
+   * Wait for a response from the PyTorch maintainers. They will review your proposal and
+     determine whether a tutorial on the proposed topic is desirable. **Do not submit a PR with
+     your tutorial** before you receive an approval.
+     
+1. **Adhere to writing and styling guidelines:**
+   * Once approved, follow the guidelines outlined in [CONTRIBUTING.md](https://github.com/pytorch/tutorials/blob/main/CONTRIBUTING.md)
+     and use the provided [template](https://github.com/pytorch/tutorials/blob/main/beginner_source/template_tutorial.py) for creating your tutorial.
+   * Link the issue where you received an approval for your tutorial in the PR.
+   * We accept tutorials in both ``.rst`` (ReStructuredText) and ``.py`` (Python)
+     formats. However, unless your tutorial involves using multiple GPU, parallel/distributed
+     training, or requires extended execution time (25 minutes or more), we prefer submissions
+     in Python file format versus ``.rst``.
+     
+1. **Maintenance:**
+   * By submitting a new tutorial, you agree to maintain it up-to-date with the latest PyTorch
```

Comment:
Is this a common expectation? My understanding is people submit tutorials to learn in public and having forever maintenance is too strong of an expectation. If something is high quality enough perhaps the core teams should maintain it instead and if it's not high quality then we should remove it. 

I'm not sure requiring yearly updates makes sense, if the content is good it's unlikely to need such updates